src/mockchain/lib/Convex/MockChain.hs

```diff
@@ -495,6 +494,18 @@ instance (Monad m, C.IsAlonzoBasedEra era, C.IsEra era) => MonadBlockchain era (
     toLedgerStakeCredentials creds' = Set.fromList $ C.toShelleyStakeCredential <$> Set.toList creds'
     fromLedgerStakeAddress = C.makeStakeAddress nid . C.fromShelleyStakeCredential
   queryStakePools = MockchainT (asks npStakePools)
+  queryStakeVoteDelegatees stakeCreds = MockchainT $ C.conwayEraOnwardsConstraints @era C.conwayBasedEra $ do
```

Comment:
It would be nice to add a test for this

Reply:
Done! https://github.com/j-mueller/sc-tools/pull/265/files#diff-730dd11d4ff535028c220679a3064b6359ec940b03cc11dde62f312566298887R540